src/Adapter/Driver/Mysqli/Statement.php

```diff
@@ -206,7 +206,8 @@ public function prepare($sql = null)
         $this->resource = $this->mysqli->prepare($sql);
         if (!$this->resource instanceof \mysqli_stmt) {
             throw new Exception\InvalidQueryException(
-                'Statement couldn\'t be produced with sql: ' . $sql,
+                ' Error #' . $this->mysqli->errno . ': ' . $this->mysqli->error .
```

Comment:
Please use `sprintf`.

Reply:
Sorry for the late update. Should be sorted now.